app/services/idv/actions/redo_address_action.rb

```diff
@@ -6,6 +6,7 @@ def self.analytics_submitted_event
       end
 
       def call
+        mark_step_incomplete(:address)
```

Comment:
Is this for the unsupervised flow? I know that Team Ada has been working to remove the FlowStateMachine behaviors from many of these steps, so might be good to have someone from their team review / looped in, if not already.

Reply:
Thanks for the ping. I'm wondering if this change was intended to be in the in-person version of this action.

Reply:
It was, but is also completely uneccessary. It's been removed.